### PR TITLE
Add centos6 gpu ci build.

### PR DESCRIPTION
**Pull request by yuslepukhin**
**Description**:  Configure centos6 gpu CI build

**Motivation and Context**
Need CI
This PR has a dependency https://github.com/microsoft/onnxruntime/pull/2159

### CODE REVIEW

File: tools/ci_build/github/linux/run_dockerbuild.sh [hunk 1]
```diff
@@ -48,6 +48,10 @@ elif [ $BUILD_OS = "centos7" ]; then
     IMAGE="centos7"
     DOCKER_FILE=Dockerfile.centos
     docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} -f $DOCKER_FILE .
+elif [ $BUILD_OS = "centos6" ]; then
+    IMAGE="centos6gpu"
+    DOCKER_FILE=Dockerfile.centos6_gpu
```
Comment (HectorSVC) [inline tools/ci_build/github/linux/run_dockerbuild.sh]: >Dockerfile.centos6_gpu [](start = 16, length = 22)

where is this file? check-in already?

File: tools/ci_build/github/azure-pipelines/centos6-gpu-ci-pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    - task: CmdLine@2
+      inputs:
+        script: |
+          docker build -t onnxruntime-centos6gpu --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=3.6 -f Dockerfile.centos6_gpu .
+        workingDirectory: $(Build.SourcesDirectory)/tools/ci_build/github/linux/docker
+    - task: CmdLine@2
+      inputs:
+        script: |
+          docker run --rm --volume $(Build.SourcesDirectory):/onnxruntime_src --volume $(Build.BinariesDirectory):/build -e NIGHTLY_BUILD onnxruntime-centos6gpu /usr/bin/python36 /onnxruntime_src/tools/ci_build/build.py --build_dir /build --config $(BuildType) --skip_submodule_sync  --parallel --build_shared_lib --use_openmp --cmake_path /usr/bin/cmake --ctest_path /usr/bin/ctest --use_automl --build_wheel
+        workingDirectory: $(Build.SourcesDirectory)
```
Comment (HectorSVC) [inline tools/ci_build/github/azure-pipelines/centos6-gpu-ci-pipeline.yml]: why do you need to update the run_dockerbuild.sh which is not used for your case? Please refer to other linux build pipeline.

File: tools/ci_build/github/linux/run_dockerbuild.sh (hunk 1 above)
Comment (yuslepukhin) [reply tools/ci_build/github/linux/run_dockerbuild.sh]: This is a different PR.

---
In reply to: [336321806](https://github.com/microsoft/onnxruntime/pull/2165#discussion_r336321806) [](ancestors = 336321806)

File: tools/ci_build/github/azure-pipelines/centos6-gpu-ci-pipeline.yml (hunk 2 above)
Comment (yuslepukhin) [reply tools/ci_build/github/azure-pipelines/centos6-gpu-ci-pipeline.yml]: Not sure I understand the comment.

---
In reply to: [336322149](https://github.com/microsoft/onnxruntime/pull/2165#discussion_r336322149) [](ancestors = 336322149)

**Pull request closed without merging**